bag.h

```diff
@@ -1,86 +1,90 @@
 #include "item.h"
 
-class Bag{
-    private: //these need to be private so the calling function cannot manipulate them
-        int n;
-        Item item_des[200];
-    public: //class functions used to manipulate data
-        
-       
-        int getN(){return n;}
-        Bag(){n=0;}
-
-        bool add(string item);
-        void display();
-        void remove(string item);
-        void clear(){n = 0;}
-        bool isEmpty();
-        bool contains(string item);
-        int getFrequencyOf(string item);
-        
+class Bag
+{
+  private: //these need to be private so the calling function cannot manipulate them
+    int n;
+    Item item_des[200];
+
+  public: //class functions used to manipulate data
+    int getN() { return n; }
+    Bag() { n = 0; }
+
+    bool add(string item);
+    void display();
+    void remove(string item);
+    void clear() { n = 0; }
+    bool isEmpty();
+    bool contains(string item);
+    int getFrequencyOf(string item);
 };
 
-bool Bag::add(string item){
-    Item("item");
+bool Bag::add(string item)
+{
     item_des[n] = item;
     n++;
     return true;
 }
 
-void Bag::display(){
-    for (int i = 0; i < n; i++){
+void Bag::display()
+{
+    for (int i = 0; i < n; i++)
+    {
         cout << item_des[i].getName() << endl;
     }
 }
 
-void Bag::remove(string remove_item){
-    int count = 0;
-    while(!(item_des[count].getName() == remove_item) && count < n){
-        count++;
+void Bag::remove(string remove_item)
```

Comment:
I refactored this just slightly in a way that makes a bit more sense in how I would go about it for readability